Update server-requirements.md (#2593)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md

## Back end databases that will work with Skype for Business Server 2015
<a name="DBs"> </a>

> > [!IMPORTANT]

Reviewer: ```suggestion
> [!NOTE]
```

> > SQL back end must be local to the SFB front ends (not across a low speed link). It is not supported to share SQL back ends between two or more pools.

Reviewer: ```suggestion
> For a well-connected SQL back end, the connection to the SFB front end should be local, and not across a low speed link. Sharing SQL back ends between two or more pools is not supported.
```
